.github/workflows/vib-ghost.yaml

```diff
@@ -0,0 +1,36 @@
+name: 'vib'
+on: # rebuild any PRs and main branch changes
```

Comment:
I already have the PR ready for fixing inconsistencies. Ghost will **not** be included, so there is a need to make the inconsistency related changes here.

Reply:
This does no longer apply after https://github.com/bitnami/charts/pull/9780. Marking this as resolved.